ec2/helpers.go

```diff
@@ -50,3 +50,21 @@ func GetEC2InstanceTags(client ec2iface.EC2API, instances []*string) (ec2Tags ma
 
 	return ec2Tags, nil
 }
+
+// GetEC2InstanceAttributes accepts any number of instance strings and returns a populated InstanceAttributes{} object for each instance
+func GetEC2InstanceAttributes(client ec2iface.EC2API, instances []*string) (ec2Attributes map[string]map[string]string, err error) {
```

Comment:
Retrieves instance attributes. Currently it only does `VpcId`. TBH i kinda hate this and am wondering if we should just transmute fields on the `ec2.Instance` struct into a map if the value is a non-compound type. i.e. strings or numeric types